app/screens/latex/latex.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    BackHandler,
+    ScrollView,
+    Text,
+    View,
+    Platform,
+} from 'react-native';
+import MathView from 'react-native-math-view';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import {popTopScreen} from '@actions/navigation';
+import {splitLatexCodeInLines} from '@utils/latex';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+export default class Latex extends React.PureComponent {
+    static propTypes = {
+        theme: PropTypes.object.isRequired,
+        content: PropTypes.string.isRequired,
+    };
+
+    componentDidMount() {
+        BackHandler.addEventListener('hardwareBackPress', this.handleAndroidBack);
+    }
+
+    componentWillUnmount() {
+        BackHandler.removeEventListener('hardwareBackPress', this.handleAndroidBack);
+    }
+
+    handleAndroidBack = () => {
+        popTopScreen();
+        return true;
+    };
```

Comment:
Any reason why this is needed? is it not handled by default?

Reply:
I copied this from the code screen but i think it might be a better solution to just extend the Latex screen from the code screen. I'll implement that so there is no duplicate code